submissions/kikinovk/JS_DOM_API/css/style.css

```diff
@@ -0,0 +1,108 @@
+:root {
+  --primary-font-color: #3F403F;
+  --primary-bg-color: #E6E8E6;
+  --secondory-font-color: #475841;
+  --secondory-bg-color: #CED0CE;
+  --secondory-bg-color-hover: #9FB8AD;  
+}
+* {
+  margin: 0;
+  padding: 0;
+}
+html {
+  height: 100%;
+  width: 100%;
+}
+body {
+  height: 100%;
+  width: 100%;
+  font-family: sans-serif;
+  font-size: 16px;
+  color: var(--primary-font-color);
+}
+.toggle_menu {
+  display: block;
+  position: absolute;
+  right: 1em;
+  top: 1em;
+  background-color: transparent;
+  border: none;
+}
+.toggle_menu__icon {
+  width: 40px;
+  fill: var(--secondory-font-color);
+}
+.nav {
+  position: absolute;
+  width: 300px;
+  min-height: 100%;
+  box-sizing: border-box;
+  transform: translate(-300px, 0);
+  transition: transform 0.3s ease; 
+  background: var(--secondory-bg-color); 
+  padding-top: 2em;
+}
+.open {
+  transform: translate(0, 0);
+}
+.nav__menu {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+.menu__item {
+  display: block;
+}
+.menu__button {
```

Comment:
I'd recommend adding `cursor: pointer` for this element. It helps users easily navigate your app.

Reply:
[](https://github.com/kottans/frontend-2021-homeworks/pull/79/commits/82a392144422db49c791214f65ec23a02bd5d1f4)